__tests__/app.test.js

```diff
@@ -65,10 +65,42 @@ describe('Getting a list of available api', ()=> {
   })
 
 
+  describe('test to see if input id of article exists', ()=> {
+    test('to see if object is returned', ()=> { 
+    return request(app)
+   .get(`/api/articles/2`)
+    .expect(200)
+    .then(({body}) => {
+
+
+  
+    
+
+    expect(body).toHaveProperty("article_id", expect.any(Number))
+    expect(body).toHaveProperty("title", expect.any(String)) 
+    expect(body).toHaveProperty("topic", expect.any(String))
+    expect(body).toHaveProperty("author", expect.any(String))
```

Comment:
I really like that you're keeping this test flexible by checking the properties and type of values 👍